### ISSUE TITLE
[RFC] Package Update Notifications (Subscription System)

### ISSUE THREAD

**Issue by PhillippOhlandt** [bolt/marketplace-site#53]
Hey,

I think I a nice feature would be when a user can subscribe to a package to get a notification email when a new version of the package becomes available. 
E-Mail notifications because it's the easiest right now. We could also plan a real notification system inside the Bolt store for the future, though.

**Comment by GawainLynch** [comment]
Yes… over all I have been wanting some way to notify users of this as well, my "concern" is that we'd only be mirroring what GitHub, GitLab, BitBucket, etc do already … but mostly throwing that into the "thinking" about the feature.

**Comment by GawainLynch** [comment]
Thinking about it just a bit more … any ideas (yet) on the subscription functionality (perfect world scenario)?

**Comment by PhillippOhlandt** [comment]
So:
1. User has an account in the store
2. User goes to a package detail page on the store
3. user clicks "notify me about new releases"
4. we store the "subscription in a table"
5. package gets updated via button in the store or update hook and surprisingly a new version is available
6. we send an email to the user about the new version
